### PR TITLE
feat: add Elu activations

### PR DESCRIPTION
**Pull request by WuZhuoran**
1. `Add` [Elu](http://arxiv.org/abs/1511.07289) activation function according to this [introduction](https://github.com/aleju/papers/blob/master/neural-nets/ELUs.md).
2. `Add` Elu to README.

### CODE REVIEW

File: neural_nets/activations/activations.py [hunk 1]
```diff
@@ -128,3 +128,22 @@ def grad(self, x):
...
+
+    def __str__(self):
+        return "Elu(alpha={})".format(self.alpha)
+
+    def fn(self, z):
+        _z = z * (z > 0) + self.alpha * (np.exp(z) - 1.) * (z < 0)
+        return _z
+
+    def grad(self, x):
+        return 1 if x >= 0 else self.fn(x) + self.alpha
```
Comment (ddbourgin) [inline neural_nets/activations/activations.py]:
```suggestion
        return np.where(x >= 0, np.ones_like(x), self.fn(x) + self.alpha)
```
The `else` clause doesn't know what to do with a boolean array. I'd opt for using `np.where` here.

File: neural_nets/activations/activations.py [hunk 2]
```diff
@@ -128,3 +128,22 @@ def grad(self, x):
 
     def grad2(self, x):
         return np.zeros_like(x)
+
+
+class Elu(ActivationBase):
```
Comment (ddbourgin) [inline neural_nets/activations/activations.py]:
```suggestion
class ELU(ActivationBase):
```
Capitalize ELU for consistency with other activations

**Comment by ddbourgin** [comment]
This looks great - thanks a lot! I've added some unit tests in a separate PR (#4). If you agree with the edits, feel free to commit and I'll merge to master :)

**Comment by WuZhuoran** [comment]
> This looks great - thanks a lot! I've added some unit tests in a separate PR (#4). If you agree with the edits, feel free to commit and I'll merge to master :)

Thank you! I will commit ASAP.

**Comment by WuZhuoran** [comment]
Suggestions Applied.

**Pull request merged**